docs/index.md

```diff
@@ -8,46 +8,20 @@ menuOrder: 1
 # Getting Started
 Welcome to Serverless Console. 
 
-The next-generation monitoring tool for Teams 
-building with Serverless architectures. The easiest way 
-to get started with Serverless Console is using the 
-latest version of Serverless Framework (v3.18+ required).
-**Note:** Only Node.js is supported at this time. 
+A Developer focused observability platform for
+teams building using Serverless Architectures.
```

Comment:
`building using` ... I'm guessing that's supposed to be `building` or `using`, but not both?

Reply:
Fixed awkward wording.